BridgeAppSDK/SBASharedInfoController.swift

```diff
@@ -34,14 +34,15 @@
 import UIKit
 
 public protocol SBASharedInfoController {
+    
+    var sharedAppDelegate: SBASharedAppDelegate { get }
+//    public var sharedAppDelegate: SBASharedAppDelegate {
```

Comment:
Yeah, wasn't sure how best to include this as a "here's the copy/paste implementation".